Add Web Interface App Catalog Reference Page (#331)
giantswarm/docs: src/content/reference/web-interface/_index.md

+++
title = "Web Interface Reference"

> description = "Documentation on our web interface, a visual way to create and delete clusters, create key pairs and more."

Reviewer: ```suggestion
description = "Documentation on our web user interface, a visual way to manage clusters, create key pairs and more."
```

Author: I don't think this extra word here is really needed.

date = "2019-07-05"
layout = "subsection"
weight = 10
+++

> # Giant Swarm's Web Interface

Reviewer: ```suggestion
# Web User Interface
```

I think the fact that this is for Giant Swarm is self-evident based on the context.

Author: I'd prefer `The Web Interface` or `Web Interface` here instead.

See ongoing discussion here: https://github.com/giantswarm/giantswarm/issues/6122

> Hopefully, if we did our job well then our Web Interface is so intuitive and

Reviewer 2: I'd remove the word Hopefully - for the sake of a more powerful statement.

> Hopefully, if we did our job well then our Web Interface is so intuitive and

Reviewer: ```suggestion
Hopefully, if we did our job well then our web user interface is so intuitive and
```

easy to use that this reference isn't necessary.

However, for those of you interested in having an overview of what is possible

> with our Web Interface, and how to access it, read on!

Reviewer: ```suggestion
with our web user interface, and how to access it, read on!
```

> ## What can I do with the Web Interface?

Reviewer: ```suggestion
## What can I do with it?
```

> Our Web Interface let's you do the following things. Some topics link to more

Reviewer: ```suggestion
Our web UI lets you do the following things. Some topics link to more
```

detailed information:

- View and manage clusters
- View and manage keypairs for your clusters
- View and manage organizations
- Manage your own account and password
- Learn how to get started with your kubernetes clusters by following a guide
- [Add, Remove and configure Apps on your clusters.](app-catalog/)

## Feedback {#feedback}

> We welcome your feedback our Web Interface. If you feel like sharing openly, use the GitHub repository and create an [issue](https://github.com/giantswarm/happa/issues), so other users can participate. Otherwise please use the common Giant Swarm support channels.

Reviewer: ```suggestion
We welcome your feedback to our web UI. If you feel like sharing openly, use the GitHub repository and create an [issue](https://github.com/giantswarm/happa/issues), so other users can participate. Otherwise please use the common Giant Swarm support channels.
```

Author: I think it should be 'feedback on' right? I made that change.

Reviewer 3: `on` or `for` work in this context. `to` reads a bit strangely.
